Core/GDCore/IDE/Dialogs/LayoutEditorCanvas/LayoutEditorCanvasOptions.cpp

```diff
@@ -21,7 +21,10 @@ LayoutEditorCanvasOptions::LayoutEditorCanvasOptions()
       gridG(180),
       gridB(255),
       zoomFactor(1),
-      windowMask(false) {}
+      windowMask(false),
+      viewXPosition(400),

```

Comment:
Let's initialize these to 0, otherwise this gives a false hint that these numbers are some default values - while they are not used because viewPositionValid is false.